src/Aquifer.Jobs/Configuration/ConfigurationOptions.cs

```diff
@@ -33,11 +32,35 @@ public class AnalyticsOptions
 public class EmailOptions
 {
     public required string SubjectPrefix { get; init; }
+    public required MarketingEmailOptions Marketing { get; init; }
```

Comment:
Note: This will require updating the environment variable names in QA and Prod to follow the new naming convention.

Reply:
Do we want to move it back to where it was then or just make a mention of the needed change in slack?

Reply:
I'd put it back and then put a `NotificationsOptions` at the root level named `Notifications` with the three template IDs in it.